src/app/spans.rs

```diff
@@ -8,6 +8,11 @@ use tokio::sync::mpsc;
 const SPAN_KIND_SERVER: i32 = 1;
 const SPAN_KIND_CLIENT: i32 = 2;
 
+/// SpanConverter converts trace_context::Span objects into OpenCensus agent
```

Comment:
This is _extremely_ nitpicky, and feel free to ignore it, but: this reads like a Go doc comment to me. I think the preferred idiom in rust is:
- the first sentence of the doc comment should be a summary on a separate line
- RustDoc comments generally _don't_ start with the name of the type or function they document

See also: https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#appendix-a-full-conventions-text